resources/packaging/archlinux/PKGBUILD.in

```diff
@@ -16,9 +16,9 @@ url="https://github.com/Eloston/ungoogled-chromium"
 license=('BSD')
 depends=('gtk3' 'nss' 'alsa-lib' 'xdg-utils' 'libxss' 'libcups' 'libgcrypt'
          'ttf-font' 'systemd' 'dbus' 'libpulse' 'pciutils' 'json-glib'
-         'desktop-file-utils' 'hicolor-icon-theme' 'libevent')
-makedepends=('python2' 'gperf' 'yasm' 'mesa' 'ninja' 'git' 'libva'
-             'clang' 'llvm' 'lld' 'quilt')
+         'desktop-file-utils' 'hicolor-icon-theme')
+makedepends=('python2' 'gperf' 'yasm' 'mesa' 'ninja' 'nodejs' 'git' 'clang'
```

Comment:
I don't understand why the `nodejs` is needed here. I already disabled WebUI optimization in `common` base bundle (`optimize_webui=false`) and removed the symlink to the nodejs binary from the PKGBUILD. Is there still a reference to nodejs somewhere?

Reply:
You are right, it can be removed.